src/helpers/validation.js

```diff
@@ -0,0 +1,13 @@
+export const checkLogin = (login) => {
+  return (login.length >= 6 && typeof login === 'string') ? true : false;
```

Comment:
remember what the && operator returns?

Reply:
and use regular expressions and the test method to test the string. Does it match a regular expression?